docs/manual/comparison_with_pandas.rst

```diff
@@ -0,0 +1,246 @@
+
+Select rows on multiple conditions, using ``OR``
+``df.loc[(df['A'] > 3) | (df['B']<5)]``                   ``DT[(f.A>3) | (f.B<5), :]``
+
+Select rows on multiple conditions, using ``AND``
+``df.loc[(df['A'] > 3) & (df['B']<8)]``                  ``DT[(f.A>3) & (f.B<8), :]``
+
+Select a single column by column name
+       ``df['A']``                                     ``DT['A']``
+
+Same as above
+       ``df.loc[:, 'A']``                              ``DT[:, 'A']``
+
+Select a single column by position
+``df.iloc[:, 1]``                                       ``DT[1]`` or ``DT[:, 1]``
+
+Select multiple columns by column names
+``df.loc[:, ["A", "B"]]``                              ``DT[:, ["A", "B"]]``
+
+Select multiple columns by position
+``df.iloc[:, [0, 1]]``                                ``DT[:, [0, 1]]``
+
+Select multiple columns by slicing
+``df.loc[:, "A":"B"]``                                 ``DT[:, "A":"B"]``
+
+Same as above, by position
+``df.iloc[:, 1:3]``                                      ``DT[:, 1:3]``
+
+Select columns by boolean
+``df.loc[:, [True,False,False,True]]``                ``DT[:, [True,False,False,True]]``
+
+Select multiple rows and columns
+``df.loc[2:5, "A":"B"]``                              ``DT[2:5, "A":"B"]``
+
+Same as above, by position
+``df.iloc[2:5, :2]``                                    ``DT[2:5, :2]``
+
+Select a single value (returns a scalar)
+``df.at[2, 'A']`` or ``df.loc[2, 'A']``                 ``DT[2,"A"]``
+
+Same as above, by position
+``df.iat[2, 0]``  or  ``df.iloc[2, 0]``                 ``DT[2, 0]``
+
+Select a single value, return as Series                Returns a Frame
+``df.loc[2, ["A"]]``                                  ``DT[2, ["A"]]``
+
+Same as above, by position
+``df.iloc[2, [0]]``                                  ``DT[2, [0]]``
+=================================================  ============================================
+
+
+In ``pandas`` the index numbers are returned :
+
+.. code-block:: python
+
+    # pandas
+    df.loc[df['A']>3]
+
+    	A	B	C	D
+    3	4	7	10	9
+    4	5	8	11     -1
+
+This is not the case in ``datatable`` :
+
+.. code-block:: python
+
+    DT[f.A>3, :]
+
+        A	B	C	D
+    0	4	7	10	9
+    1	5	8	11     −1
+
+In datatable, the index numbers for the filtered rows are not returned, instead new indices are generated.
+
+
+In ``pandas``, the index can be numbers, or characters, or intervals, or even MultiIndexes; you can subset rows on these labels.
+
+.. code-block:: python
+
+    # pandas
+    df1 = df.set_index(pd.Index(['a','b','c','d','e']))
+
+        A	B	C	D
+    a	1	4	7	5
+    b	2	5	8	7
+    c	3	6	9	2
+    d	4	7	10	9
+    e	5	8	11     -1
+
+    df1.loc["a":"c"]
+
+
+        A	B	C	D
+    a	1	4	7	5
+    b	2	5	8	7
+    c	3	6	9	2
+
+``datatable`` does not support index notation this way; yes, you could have multiple indexes or character indexes, but their purpose is for use in joins, not for subsetting data :
```

Comment:
We could think about what API is missing here, but conceptually `key` columns are supposed to be the replacement for pandas indexes.

Reply:
being able to filter on the keys, allowing non unique keys as well would be good additions to the API for keys